addons/pos_sale/report/sale_report.py

```diff
@@ -62,7 +62,9 @@ def _query(self, with_clause='', fields={}, groupby='', from_clause=''):
 
         for field in fields.keys():
             select_ += ', NULL AS %s' % (field)
+        return select_
 
+    def _from_pos(self, from_clause=''):
```

Comment:
```suggestion
    def _from_pos(self):
```

The from_clause is defined from the sale POV.  It's not used here and therefore shouldn't be given as parameter to this method.